inference-engine/tests/functional/plugin/shared/include/behavior/infer_request_config.hpp

```diff
@@ -0,0 +1,96 @@
+    SKIP_IF_CURRENT_TEST_IS_DISABLED()
+    // Create CNNNetwork from ngrpah::Function
+    InferenceEngine::CNNNetwork cnnNet(function);
+    // Load config
+    std::map<std::string, std::string> config = {{CONFIG_KEY(EXCLUSIVE_ASYNC_REQUESTS), CONFIG_VALUE(YES)}};
+    config.insert(configuration.begin(), configuration.end());
+    if (targetDevice.find(CommonTestUtils::DEVICE_MULTI) == std::string::npos &&
+        targetDevice.find(CommonTestUtils::DEVICE_HETERO) == std::string::npos) {
+        ASSERT_NO_THROW(ie->SetConfig(config, targetDevice));
+    }
+    // Load CNNNetwork to target plugins
+    auto execNet = ie->LoadNetwork(cnnNet, targetDevice, config);
+    execNet.CreateInferRequest();
+
+    if ((targetDevice == CommonTestUtils::DEVICE_HDDL) || (targetDevice == CommonTestUtils::DEVICE_GNA)) {
+        ASSERT_EQ(0u, InferenceEngine::ExecutorManager::getInstance()->getExecutorsNumber());
+    } else if ((targetDevice == CommonTestUtils::DEVICE_FPGA) || (targetDevice == CommonTestUtils::DEVICE_MYRIAD) ||
+               (targetDevice == CommonTestUtils::DEVICE_KEEMBAY)) {
+        ASSERT_EQ(2u, InferenceEngine::ExecutorManager::getInstance()->getExecutorsNumber());
+    } else if (targetDevice == CommonTestUtils::DEVICE_MULTI) {
+    } else {
+        ASSERT_EQ(1u, InferenceEngine::ExecutorManager::getInstance()->getExecutorsNumber());
+    }
+}
+
+TEST_P(InferConfigTests, withoutExclusiveAsyncRequests) {
+    ASSERT_EQ(0u, InferenceEngine::ExecutorManager::getInstance()->getExecutorsNumber());
+    // Skip test according to plugin specific disabledTestPatterns() (if any)
+    SKIP_IF_CURRENT_TEST_IS_DISABLED()
+    // Create CNNNetwork from ngrpah::Function
+    InferenceEngine::CNNNetwork cnnNet(function);
+    // Load config
+    std::map<std::string, std::string> config = {{CONFIG_KEY(EXCLUSIVE_ASYNC_REQUESTS), CONFIG_VALUE(YES)}};
+    config.insert(configuration.begin(), configuration.end());
+    if (targetDevice.find(CommonTestUtils::DEVICE_MULTI) == std::string::npos &&
+        targetDevice.find(CommonTestUtils::DEVICE_HETERO) == std::string::npos) {
+        ASSERT_NO_THROW(ie->SetConfig(config, targetDevice));
+    }
+    // Load CNNNetwork to target plugins
+    auto execNet = ie->LoadNetwork(cnnNet, targetDevice, config);
+    execNet.CreateInferRequest();
+
+    if ((targetDevice == CommonTestUtils::DEVICE_GNA) || (targetDevice == CommonTestUtils::DEVICE_HDDL)) {
+        ASSERT_EQ(0u, InferenceEngine::ExecutorManager::getInstance()->getExecutorsNumber());
+    } else if (targetDevice == CommonTestUtils::DEVICE_MULTI) {
+    } else if (targetDevice == CommonTestUtils::DEVICE_MYRIAD) {
+        ASSERT_EQ(2u, InferenceEngine::ExecutorManager::getInstance()->getExecutorsNumber());
+    } else {
+        ASSERT_EQ(1u, InferenceEngine::ExecutorManager::getInstance()->getExecutorsNumber());
+    }
+}
+
+using InferConfigInTests = BehaviorTestsUtils::BehaviorTestsBasic;
+
+TEST_P(InferConfigInTests, CanInferWithConfig) {
```

Comment:
Is that the right name? As I understand it, "in" means "incorrect". [For example](https://github.com/openvinotoolkit/openvino/pull/618/files#diff-3e4e13a04fee0320c2ca4c30d26e4d4aR63) `inconfigs `:  

```
    const std::vector<std::map<std::string, std::string>> inconfigs = {
            {{VPU_MYRIAD_CONFIG_KEY(PROTOCOL), "BLUETOOTH"}},
            {{VPU_MYRIAD_CONFIG_KEY(PROTOCOL), "LAN"}},
    ...
    }
```

But in this test case I see:
`ASSERT_NO_THROW(req.Infer());`

Reply:
@nikita-kud Yes, you are right, this is not correct, the names were taken from old tests.

Reply:
I'm preparing PR.  Can I change it like this:  
```
using InferConfigInTests = BehaviorTestsUtils::BehaviorTestsBasic;

TEST_P(InferConfigInTests, CanInferWithConfig) {
```
->  
`TEST_P(InferConfigTests, CanInferWithConfig)`  

and in file `infer_request_config.cpp` :
`const std::vector<std::map<std::string, std::string>> Inconfigs = {`
->
`const std::vector<std::map<std::string, std::string>> configs = {`

Reply:
These names are already in use, it would be better like this: 
TEST_P(InferConfigInTests, CanInferWithConfig) -> TEST_P(InferWithConfigTests, CanInferWithConfig)
const std::vector<std::map<std::string, std::string>> Inconfigs  ->  const std::vector<std::map<std::string, std::string>> Fullconfigs